mappings/net/minecraft/server/ChunkMap.mapping

```diff
@@ -0,0 +1,63 @@
+CLASS net/minecraft/class_167 net/minecraft/server/ChunkMap
+	FIELD field_2129 players Ljava/util/List;
+	FIELD field_2130 chunkMapping Lnet/minecraft/class_101;
+	FIELD field_2131 chunksToUpdate Ljava/util/List;
+	FIELD field_2132 server Lnet/minecraft/server/MinecraftServer;
+	FIELD field_2133 dimensionId I
+	FIELD field_2134 viewRadius I
```

Comment:
In GameOptions it'scalled viewDistance, is this functionally different from that value?